packages/components/src/form-token-field/style.scss

```diff
@@ -134,20 +131,19 @@
 .components-form-token-field__token-text {
 	border-radius: $radius-x-small 0 0 $radius-x-small;
 	padding: 0 0 0 8px;
+	line-height: 24px;
```

Comment:
Moving here because it's overwritten for the remove token button on line 143 anyway.